### DISCUSSION TITLE
Egui integration?

### DISCUSSION THREAD

**Discussion by erikpa1** [FyroxEngine/Fyrox#326] [Ideas]
Hi, 

it could be nice, if fyrox had integration with  egui or imgui.

nothing bad agains fyrox ui as it is now, but egui has a lot of features which are also good for poeple which don't use rust only for games, but also for 3D editors, and digital twin softwares,.... (like me :D :D )

https://www.egui.rs/#demo
https://github.com/emilk/egui

**Comment by mrDIMAS** [comment]
Hi, did you know that the editor of the engine is made using fyrox-ui?
